src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -484,8 +473,7 @@ static void rr_handover(grpc_exec_ctx *exec_ctx, glb_lb_policy *glb_policy,
     grpc_lb_policy_pick(exec_ctx, glb_policy->rr_policy, &pp->pick_args,
```

Comment:
What happens if this returns true (i.e., if the pick completes immediately)?  In that case, won't we fail to invoke (or clean up) the closure?

If it's somehow not possible for this to return true, then we should probably have an assert and a comment explaining why.